app/assets/javascripts/charts.js

```diff
@@ -1,4 +1,5 @@
 $.fn.flot_pie = function() {
+  tfm.tools.deprecate('$.flot_pie()', 'react DonutChart component', '2.6')
```

Comment:
I think there's a few other functions that are only used by `flot_pie`s  which we can deprecate and clean up (e.g. `get_pie_chart`, `expand_chart`, `search_on_click` and maybe a few others)

Reply:
I thought all of them are just internally used functions. `search_on_click` is IMHO, so noone should use it as standalone, so deprecating the pie_chart should mean deprecating this as well?